### PR TITLE
Implemented regenerate all

### PR DESCRIPTION
**Pull request by judovana**

### CODE REVIEW

**Comment by judovana** [comment]
On curretn setup,
http://thor-old1.brq.redhat.com:8888/misc/regenerateAllJobs
returns
org.fakekoji.jobmanager.ManagementException: Unknown build platform: w2019.x86_64.vagrant

Which looks correct....

**Comment by judovana** [comment]
after removing...
org.fakekoji.jobmanager.ManagementException: Unknown build platform: w2019.x86_64

We really needs that combobox :)

**Comment by judovana** [comment]
after few more hacking inprojects jsons it seems to work.
As a fun thing at the end the regenerated job is without package name. ( I guess it is related to recent change which changed behavior of jdkMajor->packageName

**Comment by judovana** [comment]
todo - disable ssh api for tests
todo - disable ssh api also for regeneration, and at the end enforce reload all?
todo - if one by one, maybe to "ping" the ssh jenkins api first? (I noted that first call of  a day soemtimes fail)

**Comment by judovana** [comment]
jdkMajor->packageName == https://github.com/judovana/jenkins-scm-koji-plugin/commit/571a1ed013ad90d445eeb780ff1761fdd5a2f0b0 ??

**Comment by judovana** [comment]
Decided to go with ping

File: fake-koji/src/main/java/org/fakekoji/api/http/rest/OToolService.java [hunk 1]
```diff
@@ -71,16 +72,42 @@ public OToolService(AccessibleSettings settings) {
...
+                    jenkinsJobUpdater,
+                    settings.getLocalReposRoot(),
+                    settings.getScriptsRoot()
+            );
+
             path(MISC, () -> {
                 path(REGENERATE_ALL_JOBS, () -> get(ctx -> {
-                    // TODO: implement
+                    JobUpdateResults r = new JobUpdateResults();
+                    try{
```
Comment (oklinov) [inline fake-koji/src/main/java/org/fakekoji/api/http/rest/OToolService.java]: please use OToolHandlerWrapper so you don't need to catch exceptions...see JDK_TEST_PROJECT endpoint below

Comment (judovana) [reply fake-koji/src/main/java/org/fakekoji/api/http/rest/OToolService.java]: Thanx! Done!

**Pull request merged**